theme/theme.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2015 ThoughtWorks, Inc.
+
+// This file is part of getgauge/html-report.
+
+// getgauge/html-report is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// getgauge/html-report is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with getgauge/html-report.  If not, see <http://www.gnu.org/licenses/>.
+
+package theme
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/getgauge/common"
+	"github.com/getgauge/html-report/env"
+)
+
+const (
+	reportThemeProperty = "GAUGE_HTML_REPORT_THEME_PATH"
+)
+
+var TemplateBasePath string
+
+func GetDefaultThemePath() string {
+	if TemplateBasePath == "" {
+		dir, _ := env.GetCurrentExecutableDir()
+		TemplateBasePath = filepath.Join(dir, "..", "..", "themes")
```

Comment:
Getting the following error while running `gauge specs`
```
2017/04/06 10:49:00 open $HOME/.gauge/plugins/html-report/themes/default/views/partials.tmpl: no such file or directory
```

I think the TemplateBasePath should be `filepath.Join(dir, "..", "themes")`.